networking/simple-webserver.c

```diff
@@ -0,0 +1,48 @@
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <netinet/in.h>
+#include <string.h>
+
+/*
+  This is a very simple implementation of a web server using C. It is my first contributuion to pdhoot's repository in GitHub.
+  Creates a socket and returns a simple HTML response.
+*/
```

Comment:
If you are implementing a Web-Server, HTTP protocol should be the way you do it. In an HTTP, first a client connects to the server. After the connection is established, the client sends certain HTTP headers, which the server should process and then respond accordingly. Have a look at [this](http://www.garshol.priv.no/download/text/http-tut.html).